src/client/styles/adminpanel.css

```diff
@@ -0,0 +1,129 @@
+.modal {
+    position: fixed; /* фиксированное положение */
```

Comment:
@VRatick 
[Read this.](https://ru.wikipedia.org/wiki/%D0%9A%D0%BB%D0%B5%D0%BF%D1%82%D0%BE%D0%BC%D0%B0%D0%BD%D0%B8%D1%8F)